userguide/sharing.rst

```diff
@@ -2122,61 +2110,54 @@ file to be created is appended to the pool or dataset name.**
+   |                    |                |                                                                                                                          |
+   +====================+================+==========================================================================================================================+
+   | Extent name        | string         | Enter the extent name. If the :guilabel:`Extent size` is not *0*, it cannot be an existing file within the               |
+   |                    |                | pool or dataset.                                                                                                         |
+   +--------------------+----------------+--------------------------------------------------------------------------------------------------------------------------+
+   | Extent type        | drop-down menu | *File* shares the contents of an individual file. *Device* shares an entire device.                                      |
+   +--------------------+----------------+--------------------------------------------------------------------------------------------------------------------------+
+   | Path to the extent | browse button  | Only appears when *File* is selected. Browse to an existing file. Create a new file by browsing to a dataset and         |
+   |                    |                | appending the file name to the path. Extents cannot be created inside a jail root directory.                             |
+   +--------------------+----------------+--------------------------------------------------------------------------------------------------------------------------+
+   | Extent size        | integer        | Only appears when *File* is selected. Entering *0* uses the actual file size and requires that the file already exists.  |
+   |                    |                | Otherwise, specify the file size for the new file.                                                                       |
+   +--------------------+----------------+--------------------------------------------------------------------------------------------------------------------------+
+   | Device             | drop-down menu | Only appears when *Device* is selected. Select the unformatted disk, controller, zvol, zvol snapshot, or HAST device.    |
+   |                    |                |                                                                                                                          |
+   +--------------------+----------------+--------------------------------------------------------------------------------------------------------------------------+
+   | Logical block size | drop-down menu | Only override the default if the initiator requires a different block size.                                              |
```

Comment:
This should probably have an introductory sentence about exactly what the "logical" block size means.